src/output-filestore.c

```diff
@@ -194,10 +194,7 @@ static int OutputFilestoreLogger(ThreadVars *tv, void *thread_data, const Packet
 
     SCLogDebug("ff %p, data %p, data_len %u", ff, data, data_len);
 
-    char base_filename[PATH_MAX] = "";
-    snprintf(base_filename, sizeof(base_filename), "%s/file.%u",
-            ctx->tmpdir, ff->file_store_id);
-    snprintf(filename, sizeof(filename), "%s", base_filename);
+    snprintf(filename, sizeof(filename), "%s/file.%u", ctx->tmpdir, ff->file_store_id);
```

Comment:
it looks like this can be optimized further. In the code directly below we see that the file descriptor can be cached, and if it is we don't need the filename at all (well I guess only in the error path, we could construct it then)

Reply:
Indeed